### ISSUE TITLE
Fix padding issues with RNN model

### ISSUE THREAD

**Issue by Michael0x2a** [Michael0x2a/nlp-capstone#2]
Ideas to explore:

- [ ] Apply a mask before computing loss
- [x] Look into using dynamically-resizing RNNs, and having the width vary per batch

Potential issues:

- [ ] Comment width can vary widely; how do we handle outliers?

**Comment by Michael0x2a** [comment]
Well, I attempted looking into using dynamically-resizing RNNs, and feeding in sequence length, and other such things, and I got basically the same results, except that each epoch took 3 times longer to run...

I'm a bit stumped as to how I'd apply a mask before computing loss, given that I'm comparing the softmax cross entropy between the output labels (and so a mask isn't applicable).
